packages/ckeditor5-engine/src/controller/datacontroller.js

```diff
@@ -439,14 +439,14 @@ export default class DataController {
 	}
 
 	/**
-	 * Event fired once data initialisation has finished.
+	 * Event fired once data initialization has finished.
 	 *
 	 * @event ready
 	 */
 
 	/**
 	 * Event fired after {@link #init init() method} has been run. It can be {@link #listenTo listened to} to adjust/modify
```

Comment:
```suggestion
	 * Event fired after the {@link #init `init()` method} was run. It can be {@link #listenTo listened to} in order to adjust or modify
```